src/qibolab/_core/instruments/emulator/operators.py

```diff
@@ -13,6 +18,16 @@ def transmon_destroy(n: int) -> Qobj:
     return destroy(n)
 
 
+def identity(n: int) -> Qobj:
+    """Identity operator for n levels system."""
+    return qeye(n)
+
+
+def tensor_product(*args) -> Qobj:
+    """Tensor product of a list of operators using Qutip."""
+    return tensor(*args)
```

Comment:
It is also possible to declare them just as aliases
```suggestion
identity = qeye
"""Identity operator for n levels system."""

tensor_product = tensor
"""Tensor product of a list of operators using Qutip."""
```

You could even reproduce the type hint, just using an inline function type hint (i.e. [`Callable`](https://docs.python.org/3/library/collections.abc.html#collections.abc.Callable) - refer to the [`typing` docs](https://docs.python.org/3/library/typing.html#annotating-callable-objects) for usage explanation).
But the type should be inferred from the original object. So, it should be redundant (in which case, better to avoid, for simplicity).